combat-trainer.lic

```diff
@@ -189,12 +191,46 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
-
+      if @necro
```

Comment:
okay, so in my yaml I am creating a thanatology hash with two pairs as such:

    thanatology:
      necro: true
      ritual_type: dissect

I'm calling them into `combat-trainer.lic` with the following lines:

    def main(combat_vars)
      setup

      necro = combat_vars.thanatology['necro']
      ritual_type = combat_vars.thanatology['ritual_type']

then initializing them in the `LootProcess` class and making them an instance variable like this:

    def initialize(necro:false, ritual_type:preserve)
      @ritual_type = ritual_type
      @necro = necro

I've also added them to `LootProcess.new` as:

    combat_processes = [
      LootProcess.new(necro: necro, ritual_type: ritual_type)

Then, when I try the `echo("necro variable value: #{necro}") if $debug_mode` I get an undefined/initialized variable error.

I'm sorry for the trouble, but I'm not getting why it's not bringing it in.